src/main/java/org/beanpod/switchboard/dto/mapper/StreamMapper.java

```diff
@@ -15,5 +17,9 @@
 
   StreamDTO toDto(StreamEntity streamEntity);
 
+  List<StreamDTO> toDto(List<StreamEntity> streamEntityList);
+
   StreamModel toModel(StreamDTO streamDto);
+
+  List<StreamModel> toModel(List<StreamDTO> streamDtoList);
```

Comment:
Can rename this  to `toModels` since it's a list?

Reply:
Renamed it to `toModelList`